sprint-planning/readme.md

```diff
@@ -1,29 +1,72 @@
+     - Do NOT pre assigns stories to the future sprint. This is the purpose of the sprint planning.
+  - During the meeting: 
+    - Describes to the team the stories that they propose for the sprint.
+-  **All team members**:
+   - Listen to the product owner story description.
+   - Ask questions to make sure everyone understands each story properly.
+   - [Estimate](./estimation/readme.md) the effort for each backlog item, as a team. 
+   - Split each story into tasks.
+   - (Optional) self assign first task to team members.
+
+*(\*) some teams find useful to define a **[Definition of ready](https://www.agilealliance.org/glossary/definition-of-ready)** that describes the list of things that needs to be done in each story before the **product owner** can propose it for a **print**. The list proposed here is the classic minimal definition of ready.*
 
 ## Impact
 
-What positive impact has been observed as a result of this practice?
+**Sprint planning key benefits:**
+- Everyone participates, the entire team is aware of the scope of the sprint.
+- Each team member takes responsibility in the sprint scope by participating in the stories discovery, sign up, prioritization and estimation.
+- Creates a platform to communicate, discover and discuss dependencies.
 
 ## Measures
 
-How might one measure the impact of this ceremony positive or negative?
+- How many stories needed more work before being presented (and were rejected because not clear enough)?
+- How realistic was the estimation of effort?
+- Team satisfaction (can be assessed during the retrospective): does everyone in the team feel included in sprint planning?
 
 ## Facilitation Guidance
 
-How might one go about running the ceremony? Is there a general pattern that the ceremony should follow?
+**Prior to the meeting:**
+- Set sprint goal.
+- Make sure the backlog is prioritized.
+- Make sure each story that is a candidate for next sprint is [ready](https://www.agilealliance.org/glossary/definition-of-ready).
+
+**During the meeting:**
+- Confirming team capacity. This should be done using the average velocity (number of points achieved per sprint) and removing the project vacations/holidays/off days. If it helps, you can get the daily average velocity, divide by the number of team members and multiply by number of off days.
+- Timebox (for instance: half of the meeting for story selection, half of the meeting for task splitting).
+- Agree on how much capacity needs to be "saved" for bug fixing (might depend on the sprint).
+- Ensure everyone understands each story that is selected for the sprint.
+- Ensure everyone participates in story effort estimation.
+
+**Other considerations:**
+- Take into account off days (vacations, national holidays, unavailability).
+- If the amount of work in a Sprint is is too much to handle, you might want to split into different workstreams. This might require thinking about [scrum of scrums](./../scrum-of-scrums/readme.md) and all related ceremonies.
```

Comment:
I would suggest breaking stories into smaller stories to complete in the sprint based on teams velocity. The scrum of scrum is good when the team has more than 10-12 people so making team smaller improves efficiency in some areas.